trestlebot/cli/utils.py

```diff
@@ -35,3 +41,16 @@ def run_bot(pre_tasks: List[TaskBase], kwargs: Dict[Any, Any]) -> BotResults:
         ),
         dry_run=kwargs.get("dry_run", False),
     )
+
+
+def get_component_title(product_name: str, cac_path: str) -> str:
```

Comment:
This function is for CaC content specially, utils module is used for common functions for cli commands. Since there are following work based on this to attach component props, how about moving to a transformation workflow?

Reply:
Sounds good. How about move it to a new file trestlebot/transformers/cac_transformer.py?